rickshaw-run

```diff
@@ -979,10 +991,6 @@ sub source_container_image {
         push(@local_images, $userenv_image);
     }
     my $x = 0;
-    while ($x < $i) {
-        printf "Processing stage %d (%s)...\n\tReady\n", $x + 1, $workshop_args[$x]{'tag'};
-        $x++;
-    }
```

Comment:
I'm pretty sure I added this for a reason, not sure we should get rid of it.

Reply:
I think I was just looking for cleaner output, but I'll put it back.